esx_service/vmdk_ops.py

```diff
@@ -1810,18 +1855,30 @@ def handleVmciRequests(port):
 
         client_socket = c # Bind to avoid race conditions.
 
+        # Check the flag - if set, fail new incoming requests
+        if stopEvent.isSet():
+            svc_stop_err = 'Service is being stopped: operation declined!'
+            logging.warning(svc_stop_err)
+            send_vmci_reply(client_socket, err(svc_stop_err))
+            continue
+
         if not get_si():
             svc_connect_err = 'Service is presently unavailable, ensure the ESXi Host Agent is running on this host'
             logging.warning(svc_connect_err)
             send_vmci_reply(client_socket, err(svc_connect_err))
             continue
 
+        # Increase the in-flight ops counter
```

Comment:
comment is useless here, just takes up real estate

Reply:
Sometimes I feel it can improve the readability a little bit by adding one line comment for each separate logic segment in a big chunk of code. This is just my personal preference. I hope it's not a big concern to you:)

Reply:
I totally agree. But the comment is useless and is a distraction in the exam below
```
#increment ops counter
OpsCounter.incr()
```
IMO the comment should add something g- intent, or brief summary , etc...

Not a big deal and was a neat anyways